huxley/core/models.py

```diff
@@ -123,6 +124,8 @@ class School(models.Model):
     fees_owed = models.DecimalField(max_digits=6, decimal_places=2, default=Decimal('0.00'))
     fees_paid = models.DecimalField(max_digits=6, decimal_places=2, default=Decimal('0.00'))
 
+    modified = models.DateTimeField(default=datetime.utcnow)
```

Comment:
You'll want to use the `auto_now` and `auto_now_add` options instead: https://docs.djangoproject.com/en/1.8/ref/models/fields/#django.db.models.DateField.auto_now